packages/ionic/src/commands/package/package.ts

```diff
@@ -0,0 +1,270 @@
+import {
+  CommandLineInputs,
+  CommandLineOptions,
+  LOGGER_LEVELS,
+  OptionGroup,
+  contains,
+  validators
+} from '@ionic/cli-framework';
+import { columnar } from '@ionic/cli-framework/utils/format';
+import { sleep } from '@ionic/cli-framework/utils/process';
+import chalk from 'chalk';
+import * as Debug from 'debug';
+import * as fs from 'fs';
+import * as https from 'https';
+
+import { CommandMetadata } from '../../definitions';
+import { isSuperAgentError } from '../../guards';
+import { Command } from '../../lib/command';
+import { FatalException } from '../../lib/errors';
+
+import { fileUtils } from './utils';
+
+const debug = Debug('ionic:commands:package:build');
+const PLATFORMS = ['android', 'ios'];
+const BUILD_TYPES = ['debug', 'release', 'development', 'ad-hoc', 'app-store', 'enterprise'];
+
+interface PackageBuild {
+  job_id: number;
+  id: string;
+  caller_id: number;
+  platform: string;
+  build_type: string;
+  created: string;
+  finished: string;
+  state: string;
+  commit: any;
+  stack: any;
+  profile_tag: string;
+  automation_id: number;
+  environment_id: number;
+  native_config_id: number;
+  automation_name: string;
+  environment_name: string;
+  native_config_name: string;
+  job: any;
+}
+
+interface DownloadUrl {
+  url: string | null;
+}
+
+export class BuildCommand extends Command {
+  async getMetadata(): Promise<CommandMetadata> {
+    return {
+      name: 'build',
+      type: 'project',
+      summary: `Creates a package build on AppFlow using name parameters.`,
+      inputs: [
+        {
+          name: 'platform',
+          summary: `The platform to package (${PLATFORMS.map(v => chalk.green(v)).join(', ')})`,
+          validators: [validators.required, contains(PLATFORMS, {})],
+        },
+        {
+          name: 'type',
+          summary: `The build type (${BUILD_TYPES.map(v => chalk.green(v)).join(', ')})`,
+          validators: [validators.required, contains(BUILD_TYPES, {})],
+        },
+      ],
+      options: [
+        {
+          name: 'securityProfile',
```

Comment:
We prefer kebab cased command line option names, e.g. `--security-profile`. Options that are camel cased (like `ionic cordova run`'s `--buildConfig`) are only that way because it's actually an option of an underlying CLI.

Reply:
will change the option names